fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java

```diff
@@ -185,6 +185,21 @@ public long getTransactionId() {
         return transactionId;
     }
 
+    /**
+     * Show table names for frontend
+     * If table name could not be found by id, the table id will be used instead.
+     *
+     * @return
+     */
+    abstract Set<String> getTableNamesForShow();
+
+    /**
+     * Return the real table names by table ids.
+     * Throw MetaNofFoundException when table name could not be found.
+     * @return
+     */
```

Comment:
Could add more comment to describe the difference between getTableNamesForShow() and getTableNames().
For example, getTableNames() is used only for checking privilege?

Reply:
/(ㄒoㄒ)/~~ getTableNames is used only for old checking privilege.....